templatefitter/fit_model/model_builder.py

```diff
@@ -123,56 +130,66 @@ def __init__(
 
         # endregion
 
+        self.ncall = 0
```

Comment:
While it influences the parameter handler I feel like it belongs more to the `FitModel` (at least as far as I understand it at least) because the method that is actually called (either the nll or the chisquare) is also defined there. I'm only using the number of calls for this call cache inside the `ParameterHandler` so far but one could also imagine some other reason to check the number of calls inside the model.

However, if you think it should go inside `ParameterHandler` I will move it there, I don't feel strongly about this either.

Reply:
Let's leave it as it is now and see how this works out. :)